src/tests/sorting.spec.js

```diff
@@ -0,0 +1,48 @@
+import { test, expect } from '../fixtures/base';
+
+test.describe('sorting invertory page', () => {
+    test('sorting name A-Z', async ({ page, app }) => {
+        await app.login.navigate('/');
+        await app.login.performLogin('standard_user', 'secret_sauce');
+
+        await expect(page).toHaveURL(app.inventory.url);
+
+        await app.inventory.sortBy('az');
+
+        await expect(app.inventory.inventoryItems.nth(0).getByTestId('inventory-item-name')).toHaveText('Sauce Labs Backpack');
+    });
+
+    test('sorting name Z-A', async ({ page, app }) => {
```

Comment:
you don't need to have page here to use it in assertuins (expects). It is included to app